app/assets/src/components/ui/icons/CopyIcon.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const CopyIcon = props => {
+  return (
+    <svg
+      className={props.className}
+      viewBox="0 0 15 17"
+      version="1.1"
+      xmlns="http://www.w3.org/2000/svg"
+      xmlnsXlink="http://www.w3.org/1999/xlink"
+    >
+      <g id="Coverage-map" stroke="none" strokeWidth="1" fillRule="evenodd">
+        <g
+          id="2.6-Sample-Report-Copy-9"
```

Comment:
nit: could remove the 2.6-Sample-Report-Copy-9 and noun_copy_1801473_000000. looks a little odd